### PR TITLE
W-11493301 document datagraph UI errors

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

**Review by tomasiturralde — APPROVED** [review]
LGTM

**Review by mauro-dentoni — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 1]
```diff
@@ -1,14 +1,182 @@
...
+                },
+               "classification": "DataFetchingException"
+....
+
+An exception while fetching data error occurs when the unified schema can't connect to its underlying APIs.
+
+To troubleshoot this error, ensure that:
+
+. The underlying APIs are running.
+. The URL provided for each APIs is correct. Go to *List of APIs added* > Select an API > *API details*, and confirm the URL.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:83]:
```suggestion
. The URL provided for each API is correct. Go to *List of APIs added* > Select an API > *API details*, and confirm the URL.
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 2]
```diff
@@ -1,14 +1,182 @@
...
+. You xref:runtime-manager::to-request-vpc-connectivity.adoc[request connection] for any of your APIs that run on standalone instances that aren’t connected to Anypoint VPC.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:84]: This is not clear to me.

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 3]
```diff
@@ -1,14 +1,182 @@
...
-** xref:troubleshoot-query-logs.adoc[Troubleshoot Queries With Response Logs] +
-Anypoint DataGraph provides response logs that contain useful troubleshooting information about queries.
-** xref:troubleshoot-query-traces.adoc[Troubleshoot Query Performance with Query Tracing] +
-Query tracing in Anypoint DataGraph helps you analyze query performance by tracing every call made to the source APIs of the unified schema for a given query.
+
+Anypoint DataGraph does not translate the following parts of an API specification into an API schema:
+
+* Requests with a `Media type` value other than `application/json`
+* Responses with a `Media type` value other than `application/json`
+* Response payloads with any of the unsupported types `Schema`, `File`, `Tuple`, `Matrix`, `Nil`, or arrays of any of these types
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:109]:
```suggestion
* Response payloads with any of the unsupported types `Schema`, `File`, `Tuple`, `Matrix`, `Nil`, or arrays of any of the types
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 4]
```diff
@@ -1,14 +1,182 @@
...
+* `query params` and `headers` with types other than scalars or scalar array
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:110]: The terms scalars and array are inconsistent here.

Use scalar and array or
scalars and arrays

Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:110]:
```suggestion
* `query params` and `headers` with types other than scalars or scalar arrays
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 5]
```diff
@@ -1,14 +1,182 @@
...
+* Endpoints of required `params` of unsupported types
+* `union` types with members other than `objects` and `nils`
+* JSON schema keywords `if`, `then`, and `else`
+
+If your API specification is composed of only unsupported parts, Anypoint DataGraph does not generate an API schema and throws a *Validation failed* error.
+
+== Troubleshooting Errors When Adding, Updating, or Promoting APIs
+
+If you encounter errors when adding, updating, or promoting APIs, troubleshoot them as follows.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:119]:
```suggestion
If you encounter errors when adding, updating, or promoting APIs, troubleshoot them as follows:
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 6]
```diff
@@ -1,14 +1,182 @@
...
+
+* Two users updated an environment at the same time error
++
+If two users attempt to update an environment at the same time, DataGraph generates an error for one of the users. If you receive the error, wait for any updates to complete and again.
+
+* Issue with action performed while editing an API error
++
+If you receive such an error, try adding the API without editing to see if the edit actions created the problem.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:127]:
```suggestion
If you receive this error, try adding the API without editing to see if the edit actions created the problem. 
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 7]
```diff
@@ -1,14 +1,182 @@
...
++
+If you receive such an error, try adding the API without editing to see if the edit actions created the problem. 
+
+* Not enough capacity to deploy error
++
+DataGraph generates this error if your environment does not have the capacity available to create the unified schema. If you receive this error, request more capacity from your admin, or ask your operator to redistribute capacity within the organization.
+
+== Troubleshooting mTLS Errors
+
+If you encounter mTLS errors, troubleshoot them as follows.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:135]:
```suggestion
If you encounter mTLS errors, troubleshoot them as follows:
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 8]
```diff
@@ -1,14 +1,182 @@
...
++
+DataGraph generates this error if the private key or password does not match the certificate. Ensure the private key and certificate match or confirm the password. 
+
+* Issue with TLS error 
++
+This error occurs when trying to configure mTLS for an API that you’ve added to the unified schema. To troubleshoot this issue, add the API without configuring mTLS to determine if there is a problem with your TLS files. 
+
+== Troubleshooting Access Errors
+
+If you encounter errors when trying to access DataGraph, troubleshoot them as follows.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:151]:
```suggestion
If you encounter errors when trying to access DataGraph, troubleshoot them as follows:
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 9]
```diff
@@ -1,14 +1,182 @@
...
++
+If you encounter this error, request authorization from your admin.
+
+* Too many requests error
++
+DataGraph generates this error if you attempt too many requests. In this case, wait some time and try your request again. 
+
+== Troubleshooting Server Errors
+
+If you encounter server errors, troubleshoot them as follows.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:167]:
```suggestion
If you encounter server errors, troubleshoot them as follows:
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc (hunk 4 above)
Comment (dhirsch1) [reply modules/ROOT/pages/troubleshoot-schemas-queries.adoc:110]: I am not sure if my edit is correct.

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc (hunk 1 above)
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:83]:
```suggestion
. The URL provided for each APIs is correct. Go to *List of APIs added* > *Select an API* > *API details*, and confirm the URL.
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 10]
```diff
@@ -1,14 +1,182 @@
...
+
+* Requests with a `Media type` value other than `application/json`
+* Responses with a `Media type` value other than `application/json`
+* Response payloads with any of the unsupported types `Schema`, `File`, `Tuple`, `Matrix`, `Nil`, or arrays of any of these types
+* `query params` and `headers` with types other than scalars or scalar array
+* Endpoints of required `params` of unsupported types
+* `union` types with members other than `objects` and `nils`
+* JSON schema keywords `if`, `then`, and `else`
+
+If your API specification is composed of only unsupported parts, Anypoint DataGraph does not generate an API schema and throws a *Validation failed* error.
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:115]:
```suggestion
If your API specification is composed of only unsupported parts, Anypoint DataGraph does not generate an API schema and throws a `Validation failed` error.
```

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc [hunk 11]
```diff
@@ -1,14 +1,182 @@
...
+
+* Too many requests error
++
+DataGraph generates this error if you attempt too many requests. In this case, wait some time and try your request again. 
+
+== Troubleshooting Server Errors
+
+If you encounter server errors, troubleshoot them as follows.
+
+* Server error when creating policies, contracts, tiers, and grant types:
```
Comment (dhirsch1) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:169]:
```suggestion
* Server error when creating policies, contracts, tiers, and grant types
```

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/troubleshoot-schemas-queries.adoc (hunk 2 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-schemas-queries.adoc:84]:
```suggestion
. You xref:runtime-manager::to-request-vpc-connectivity.adoc[must request connections] for any of your APIs that run on standalone instances that aren’t connected to Anypoint VPC.
```

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**